### DISCUSSION TITLE
egzumer k5 fimware

### DISCUSSION THREAD

**Discussion by ewokkowe** [egzumer/uv-k5-firmware-custom#337] [Ideas]
is it possible in the next upgrade of the firmware that an option in the menu to turn voice on and off is added.
i have tired to do this myself but it seems i am not quite smart enough to do this , even with the instructions.
thankyou in advance if you can add this option.

**Comment by ErikS-web** [comment]
_Title makes no sense. ?? !!_

If you want **VOICE on**, you have to _do it yourself._
For this firmware it has been decided to **turn it off by default.**

Use this **_Build option_** to create your own special VOICE-ON firmware

_ENABLE_VOICE := 0 do you want to hear voices?_

Set this line to value "1"

See the options in the README.md under [user-customization.](https://github.com/egzumer/uv-k5-firmware-custom/tree/a43a3f949f32ed8ffbf0438face0edb7d093811e#user-customization)


So if you want to use it. Customize and build your firmware with the instructions

**Reply by ewokkowe** [reply]
Sorry  but I dont think you read my question properly. If you did you would see that I have tried to do what you suggested and I have failed.
I was only asking for a little help but it seems that you would rather attack instead of helping . I thought that these forums were about helping people , but it seems I was horribly wrong.  Goodbye.
